vite.config.ts

```diff
@@ -0,0 +1,42 @@
+import { resolve } from 'path';
+import { defineConfig } from 'vite';
+import vue from '@vitejs/plugin-vue';
+import checker from 'vite-plugin-checker';
+import svgLoader from 'vite-svg-loader';
+import dts from 'vite-plugin-dts';
+
+// https://vitejs.dev/config/
+export default defineConfig({
```

Comment:
Note to keep in mind recent changes: https://github.com/bethinkpl/design-system/pull/404/files#diff-1d62a367f02f5c91000e35c82b0086fae3246eca3942f9034c058b16a3780ec2R22

Reply:
@fraszczakszymon what about it? Do we need to fix anything here?

Reply:
Tested. It's fine. I forgot to resolve this comment 👍🏻